src/dbnode/persist/fs/commitlog/commit_log.go

```diff
@@ -295,6 +315,40 @@ func (l *commitLog) ActiveLogs() ([]File, error) {
 	return files, nil
 }
 
+func (l *commitLog) RotateLogs() (File, error) {
+	l.closedState.RLock()
+	defer l.closedState.RUnlock()
+
+	if l.closedState.closed {
+		return File{}, errCommitLogClosed
+	}
+
+	var (
+		err  error
+		file File
+		wg   = sync.WaitGroup{}
```

Comment:
feel free to take/leave: this can be simplified to `wg sync.WaitGroup`

Reply:
sure